vuepress/docs/next/tutorials/solution/customer-portal.md

```diff
@@ -60,132 +64,136 @@ ent ecr deploy --repo="https://github.com/entando-samples/customerportal-applica
-   Under `Credential Reset`, in the `Reset Actions` → `Update Password`  \
-   Click `Send Email` 
-
- 4. Go to Role Mapping: 
-    * Select the appropriate roles from `Available Roles` and click `Add Selected` to assign. 
-    * Choose `entandodemo-customerportal-server` from the `Client Roles` pull-down options to assign default roles. 
-    * A full administrative user will need the `realm-admin` role under `Client Roles` → `realm-management` in order to manage users in the Portal. 
-    * Check the `Effective Roles` column on the right to ensure the correct roles have been assigned.
+
+   * `cp-customer`: Assigned directly to specific projects of a single customer
+   * `cp-partner`: Assigned directly to specific projects of multiple customers
+   * `cp-support`: Read only view of all customer projects
+   * `cp-admin`: Admin access to the Customer Portal
+
+#### Create New User
+1. From the sidebar, go to `Users`
+2. Click `Add User` on the right
+3. Complete the form as needed, but note the requirements for these fields:
+    - `Username`: A unique name
+    - `Email`: Must use the same address used in Jira
+    - `User Enabled`: Set to On
+4. Click`Save`
+5. Send an email to the user to activate their account and set a new password: 
+    1. Go to the `Credentials` header 
+    2. Under `Credential Reset`, go to `Reset Actions` → `Update Password`  
+    3. Click `Send Email`
+
+#### Assign Roles to Users
+1. Click on the `Role Mapping` tab
+2. Select the appropriate role(s) from `Available Roles` and assign them to a user by clicking `Add Selected`:
+    * To assign default roles, choose `entandodemo-customerportal-server` from the `Client Roles` pull-down
+    * To manage users in the Customer Portal, a user will need full administrative privileges via the `realm-admin` role under `Client Roles` → `realm-management`
+3. Check the `Effective Roles` column on the right to ensure the correct roles have been assigned
 
 ![Entando ID Management Role Mapping](./images/cp-identity-userrole.png)
 
- 5. Under `Groups`, assign roles to groups as needed. Roles are additive.
+#### Assign Roles to Groups
+Under `Groups`, assign roles to groups as needed. Multiple roles can be assigned to a single group.
 
 ## Managing the Customer Portal
 
 As administrator for the Customer Portal, you can create and manage users, customers, projects, and subscriptions. You can assign projects to users who have activated their account on Jira and have been assigned roles in the Entando Identity Management System. 
 
 ![Customer Portal Landing Page](./images/cp-landing-page.png)
 
-* **Create a Customer or Partner**\
-Creating a `Customer` and `Partner` is a similar process. Below are the steps for adding a `Customer`. Follow the same procedure for adding a `Partner`.
+#### Create a Customer or Partner
+The processes to create customers and partners are similar. Below are the steps to add a `Customer`. Follow the analogous procedure to add a `Partner`.
```

Comment:
analogous is a kind of a hard word for many, simpler word would make it easier to understand